### DISCUSSION TITLE
Auto completion in code editor?

### DISCUSSION THREAD

**Discussion by GaetanRdn** [stackblitz/tutorialkit#317] [Q&A]
Probably the question has already been asked, but could auto completion be available in code editor ?

In my Angular tutorial, some people asked me for this after their first try.

**Comment by AriPerkkio** [comment]
This is planned and tracked on https://github.com/stackblitz/tutorialkit/issues/213.

**Reply by GaetanRdn** [reply]
You are great 🫶🏼
